diff --git a/dist/Select-49a62830.esm.js b/dist/Select-49a62830.esm.js
index bed08498732b023f350d24a62728371af6a4dace..ea7d4fbd21d2189fa48cd5dba17f086ee61f1e17 100644
--- a/dist/Select-49a62830.esm.js
+++ b/dist/Select-49a62830.esm.js
@@ -2125,7 +2125,10 @@ var Select = /*#__PURE__*/function (_Component) {
         'aria-labelledby': this.props['aria-labelledby'],
         'aria-required': required,
         role: 'combobox',
-        'aria-activedescendant': this.isAppleDevice ? undefined : this.state.focusedOptionId || ''
+        'aria-activedescendant':
+        this.state.componentHasMounted && this.isAppleDevice
+           ? undefined
+           : this.state.focusedOptionId || '',
       }, menuIsOpen && {
         'aria-controls': this.getElementId('listbox')
       }), !isSearchable && {
diff --git a/dist/Select-5dacb5ba.cjs.prod.js b/dist/Select-5dacb5ba.cjs.prod.js
index ea9203fb889d23825493d63964aad8229e758517..90f700cfa1d07fdbf169b9ca3b25af367622c37c 100644
--- a/dist/Select-5dacb5ba.cjs.prod.js
+++ b/dist/Select-5dacb5ba.cjs.prod.js
@@ -2130,7 +2130,10 @@ var Select = /*#__PURE__*/function (_Component) {
         'aria-labelledby': this.props['aria-labelledby'],
         'aria-required': required,
         role: 'combobox',
-        'aria-activedescendant': this.isAppleDevice ? undefined : this.state.focusedOptionId || ''
+        'aria-activedescendant':
+        this.state.componentHasMounted && this.isAppleDevice
+           ? undefined
+           : this.state.focusedOptionId || '',
       }, menuIsOpen && {
         'aria-controls': this.getElementId('listbox')
       }), !isSearchable && {
diff --git a/dist/Select-d63eed7b.cjs.dev.js b/dist/Select-d63eed7b.cjs.dev.js
index b3858a3b953490f3089fd60e97e1e7555b22ec15..fc39df89b770aaa97df264a60c085adeb80f697c 100644
--- a/dist/Select-d63eed7b.cjs.dev.js
+++ b/dist/Select-d63eed7b.cjs.dev.js
@@ -2149,7 +2149,10 @@ var Select = /*#__PURE__*/function (_Component) {
         'aria-labelledby': this.props['aria-labelledby'],
         'aria-required': required,
         role: 'combobox',
-        'aria-activedescendant': this.isAppleDevice ? undefined : this.state.focusedOptionId || ''
+        'aria-activedescendant':
+        this.state.componentHasMounted && this.isAppleDevice
+           ? undefined
+           : this.state.focusedOptionId || '',
       }, menuIsOpen && {
         'aria-controls': this.getElementId('listbox')
       }), !isSearchable && {
