diff --git a/dist/Select-36d15f93.cjs.prod.js b/dist/Select-36d15f93.cjs.prod.js
index e2326a0bf776ab9f9a54005fa7fea9e6aaa7b100..1704dca666c08e7be3e181dd6d3e2d7f9924d689 100644
--- a/dist/Select-36d15f93.cjs.prod.js
+++ b/dist/Select-36d15f93.cjs.prod.js
@@ -2132,7 +2132,10 @@ var Select = /*#__PURE__*/function (_Component) {
         'aria-labelledby': this.props['aria-labelledby'],
         'aria-required': required,
         role: 'combobox',
-        'aria-activedescendant': this.isAppleDevice ? undefined : this.state.focusedOptionId || ''
+        'aria-activedescendant':
+        this.state.componentHasMounted && this.isAppleDevice
+           ? undefined
+           : this.state.focusedOptionId || '',
       }, menuIsOpen && {
         'aria-controls': this.getElementId('listbox')
       }), !isSearchable && {
diff --git a/dist/Select-7eb2ef56.cjs.dev.js b/dist/Select-7eb2ef56.cjs.dev.js
index 58d420040577220d34ea9aae0f7fe757dc86e162..7498687385036a2b26a733f7854826ddf2ee2588 100644
--- a/dist/Select-7eb2ef56.cjs.dev.js
+++ b/dist/Select-7eb2ef56.cjs.dev.js
@@ -2151,7 +2151,10 @@ var Select = /*#__PURE__*/function (_Component) {
         'aria-labelledby': this.props['aria-labelledby'],
         'aria-required': required,
         role: 'combobox',
-        'aria-activedescendant': this.isAppleDevice ? undefined : this.state.focusedOptionId || ''
+        'aria-activedescendant':
+        this.state.componentHasMounted && this.isAppleDevice
+           ? undefined
+           : this.state.focusedOptionId || '',
       }, menuIsOpen && {
         'aria-controls': this.getElementId('listbox')
       }), !isSearchable && {
diff --git a/dist/Select-aab027f3.esm.js b/dist/Select-aab027f3.esm.js
index fd5726b720a2fed0e4ad5d80407a801d9e319966..5138ea8ae5c9c1b7b1d8fb24b4b5166e78df9564 100644
--- a/dist/Select-aab027f3.esm.js
+++ b/dist/Select-aab027f3.esm.js
@@ -2127,7 +2127,10 @@ var Select = /*#__PURE__*/function (_Component) {
         'aria-labelledby': this.props['aria-labelledby'],
         'aria-required': required,
         role: 'combobox',
-        'aria-activedescendant': this.isAppleDevice ? undefined : this.state.focusedOptionId || ''
+        'aria-activedescendant':
+        this.state.componentHasMounted && this.isAppleDevice
+           ? undefined
+           : this.state.focusedOptionId || '',
       }, menuIsOpen && {
         'aria-controls': this.getElementId('listbox')
       }), !isSearchable && {
